package.json

```
@@ -9,10 +11,11 @@
     "mdlint": "markdownlint-cli2 '**/*.md' '!node_modules' '!meetings/201*/*.md' '!meetings/202[0-2]*/*.md' '!meetings/2023-0[1-3]/*.md'",
     "mdlint:fix": "markdownlint-cli2-fix '**/*.md' '!node_modules' '!meetings/201*/*.md' '!meetings/202[0-2]*/*.md' '!meetings/2023-0[1-3]/*.md'",
     "//": "markdownlint most likely to fail, so run that first",
-    "test": "npm run mdlint && npm run lint && npm run check-delegates-test && npm run check-delegates"
+    "test": "npm run mdlint && npm run bad-linebreaks && npm run lint && npm run check-delegates-test && npm run check-delegates"
```

```suggestion
    "test": "npm run mdlint && npm run bad-linebreaks && npm run lint && npm run check-delegates-test && npm run check-delegates",
    "fix:all": "npm run bad-linebreaks:fix && npm run lint:fix && npm run mdlint:fix"
```
We can create a `CONTRIBUTING.md` doc and [mention to run `npm run fix:all` before making the commit](https://github.com/OnkarRuikar/markdownlint-rule-search-replace/blob/main/CONTRIBUTING.md#dev-tooling).

---

sure.  adding something to the readme would be sufficient